dipy/reconst/qtdmri.py

```diff
@@ -1070,11 +1070,11 @@ def pdf(self, rt_points):
         return eap
 
 
-def qtdmri_to_mapmri_matrix(radial_order, time_order, ut, tau):
-    """Generates the matrix that maps the qtdmri coefficients to MAP-MRI
-    coefficients. The conversion is done by only evaluating the time basis for
-    a diffusion time tau and summing up coefficients with the same spatial
-    basis orders [1].
+def _qtdmri_to_mapmri_matrix(radial_order, time_order, ut, tau, isotropic):
+    """Generates the matrix that maps the spherical qtdmri coefficients to
```

Comment:
Can you update the docstring to be in imperative mood.  `Generate` instead of `Generates`

Reply:
Done in cdf3430.